core/src/main/java/org/wildfly/channel/Channel.java

```diff
@@ -198,7 +198,7 @@ void init(MavenVersionsResolver.Factory factory) {
                 requiredChannel.init(factory);
                 requiredChannels.add(requiredChannel);
             } catch (UnresolvedMavenArtifactException | MalformedURLException e) {
-                throw new RuntimeException(String.format("Unable to resolve required channel %s:%s", groupId, artifactId, version));
+                throw new RuntimeException(String.format("Unable to resolve required channel %s:%s %s", groupId, artifactId, version));
```

Comment:
A colon character should be added after the artifactId,

Reply:
fixed.